jsonrpc/codec.go

```diff
@@ -9,6 +9,17 @@ import (
 	"github.com/0xPolygon/polygon-edge/types"
 )
 
+func init() {
+	pendingBN := PendingBlockNumber
+	PendingBlockNumberOrHash = BlockNumberOrHash{BlockNumber: &pendingBN}
+
+	latestBN := LatestBlockNumber
+	LatestBlockNumberOrHash = BlockNumberOrHash{BlockNumber: &latestBN}
+
+	earliestBN := EarliestBlockNumber
+	EarliestBlockNumberOrHash = BlockNumberOrHash{BlockNumber: &earliestBN}
+}
```

Comment:
Why not in the `var` block, but in the `init` function? Is it because you wanted to execute "instantiation" only once?

Reply:
Yes. only once.